src/ui/public/filter_manager/__tests__/filter_manager.js

```diff
@@ -114,13 +114,13 @@ describe('Filter Manager', function () {
     checkAddFilters(0, null, 3);
     expect(appState.filters).to.have.length(2);
 
-    let scriptedField = {name: 'scriptedField', scripted: true, script: 1};
+    let scriptedField = {name: 'scriptedField', scripted: true, script: 1, lang: 'painless'};
```

Comment:
I agree with you that it's a poor test. I don't feel very strongly regarding it.